src/mgr/DaemonServer.cc

```diff
@@ -1433,7 +1434,6 @@ bool DaemonServer::_handle_command(
       if (!r) {
         ss << "OSD(s) " << osds << " are safe to destroy without reducing data"
            << " durability.";
-        safe_to_destroy.swap(osds);
```

Comment:
why is this line changed?

Reply:
@liewegas At this point osds = safe_to_destroy.  I almost added an assert to show that, but decided it as overkill.  I tested it with the assert.